src/styles/foundation/_layout.scss

```diff
@@ -23,6 +24,9 @@ $layout-width-data: (
   nav: (
     base: rem($navigation-width),
   ),
+  unstable-nav: (
+    base: rem($unstable-navigation-width),
```

Comment:
This will likely cause problems with breakpoints, since the `nav-width` is a key measurement in determining where our breakpoints will apply. Check out `_breakpoints.scss` for a glimpse into just how important this unit is in setting up our breakpoints.

If nav width is changing in the new design language, we will need a way to make the breakpoints continue to work when global theming is enabled.

Reply:
We'll have to adjust the breakpoints where nav collapses during a major release. For now this means the nav likely gets hidden a little earlier than it should, since it's wider, but that's a negligible offence. The content is likely going to be squished a little too much before it gets stacked but it's just us (internally) seeing these changes anyway.